app/forms/publish/user_form.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Publish
+  class UserForm < Form
+    FIELDS = %i[
+      first_name
+      last_name
+      email
+      id
+    ].freeze
+
+    attr_accessor(*FIELDS)
+
+    validates :first_name, presence: true
+    validates :last_name, presence: true
+    validates :email, presence: true, format: { with: /\A.*@.*\z/, message: :format }
```

Comment:
Is that actually needed?

`message: :format`

Reply:
Yes, it pulls the validation error message from the translations

Reply:
Oh right - there

https://github.com/DFE-Digital/publish-teacher-training/pull/3159/commits/2804ae25e640d4c66b1996624f632994c9827e80